images/client/Containerfile.fedora

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 Michael Adam
+
+FROM registry.fedoraproject.org/fedora:36
+
+MAINTAINER Michael Adam <obnox@samba.org>
+
+LABEL org.opencontainers.image.title="Samba Client container"
+LABEL org.opencontainers.image.description="Samba Client container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
+
+RUN dnf -y install samba-client
```

Comment:
How come _Containerfile.fedora_ is not shown as rename of previous _Containerfile_?

Reply:
I think that's because git saw the added content at the same time, and determined they weren't the same file. Perhaps I should split the new labels into different commits, so that git recognizes the file move.

Reply:
@phlogistonjohn, @anoopcs9 do you want me to split these changes into different commits?

Reply:
I missed this earlier, but I'm fine with it as it is.

Reply:
This is now detected as rename in latest version.